.gitignore

```diff
@@ -10,3 +9,5 @@ local/certbot/conf/**
 local/nginx/conf.d/00-katalyst.conf
 tmpbin
 linked-peer-package
+.husky
```

Comment:
I'm not sure that this file should be ignored, that's where the configuration for the pre commit hook is stored